### DISCUSSION TITLE
How do I view the notifier history ?

### DISCUSSION THREAD

**Discussion by iamxiaojianzheng** [folke/snacks.nvim#60] [Q&A]
I can't find the keymap

**Comment by iamxiaojianzheng** [comment]
In lazyvim12, I executed the command 'Notification' or 'message' to see the contents of the notification that had disappeared

**Comment by LintaoAmons** [comment]
Have you resolved this question?

**Comment by LintaoAmons** [comment]
```lua
Snacks.notifier.show_history()
```

found it~ 

in case someone search `debug` history
